src/styles/blocks/details.scss

```diff
@@ -0,0 +1,107 @@
+.details {
+
+  &__content-wrapper {
+    @include limitContentWidth;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__heading {
+    position: relative;
+    z-index: 2;
+    text-align: center;
+
+    @include tabletBreakpoint {
+      font-size: 48px;
+    }
+
+    @include desktopBreakpoint {
+      font-size: 64px;
+    }
+  }
+
+  &__photo-row {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    padding-top: 40px;
+
+    @include tabletBreakpoint {
+      grid-template-columns: repeat(6, 1fr);
+      gap: 24px;
+      padding-top: 48px;
+    }
+
+    @include desktopBreakpoint {
+      padding-top: 48px;
+    }
+  }
+
+  &__photo {
+    position: relative;
+    z-index: 1;
```

Comment:
better move z-index to :hover - now it works weird - scaled image undergoes the static one
![Image 2022-04-14 10 39 41](https://user-images.githubusercontent.com/39308717/163337530-aa1148ff-df0a-41ce-a0dd-abe02480f72b.png)